### ISSUE TITLE
Conv2d output rows and calls are the same as the input, but channels dim increase

### ISSUE THREAD

**Issue by aurum408** [fchollet/keras#6106]
I've upgraded keras to 2.0, now convolution output dims aren't changing as expected
Here is my model, I ran it on tensorflow backend:
```
opt = Adam(lr=1e-3)
def buldStyleConv(n_w,n_h):
    inp = Input(shape=(n_w,n_h,3))
    d = Conv2D(256*3,(5,5), padding='same', activation='relu',data_format="channels_last")(inp)
    #d = LeakyReLU(0.2)(d)
    #d = Dropout(0.23)(d)
    d = Conv2D(256*3*2,5,padding='same', activation='relu',data_format="channels_last")(d)
    #d = LeakyReLU(0.2)(d)
    #d = Dropout(0.23)(d)
    d = Flatten()(d)
    d = Dense(256*3,activation='relu')(d)
    d = LeakyReLU(0.2)(d)
    d = Dropout(0.25)(d)
    d_out = Dense(8,activation='softmax')(d)

    return Model(inp,d_out)

discr = buldStyleConv(256,180)
discr.compile(opt,'categorical_crossentropy')
discr.summary()
```
The summary:
```
Layer (type)                 Output Shape              Param #   
=================================================================
input_20 (InputLayer)        (None, 256, 180, 3)       0         
_________________________________________________________________
conv2d_33 (Conv2D)           (None, 256, 180, 768)     231168    
_________________________________________________________________
conv2d_34 (Conv2D)           (None, 256, 180, 1536)    29492736  
_________________________________________________________________
flatten_17 (Flatten)         (None, 70778880)          0         
_________________________________________________________________
dense_33 (Dense)             (None, 768)               5435818060
_________________________________________________________________
leaky_re_lu_21 (LeakyReLU)   (None, 768)               0         
_________________________________________________________________
dropout_21 (Dropout)         (None, 768)               0         
_________________________________________________________________
dense_34 (Dense)             (None, 8)                 6152      
=================================================================
Total params: 54,387,910,664.0
Trainable params: 54,387,910,664.0
Non-trainable params: 0.0
_________________________________________________________________
```
Could u say what's wrong ?
Thanks

**Comment by aurum408** [comment]
sorry, it's my bad
